config/newrelic.yml

```diff
@@ -0,0 +1,66 @@
+# This file configures the New Relic Agent.  New Relic monitors Ruby, Java,
+# .NET, PHP, Python, Node, and Go applications with deep visibility and low
+# overhead.  For more information, visit www.newrelic.com.
+#
+# Generated October 28, 2022
+#
+# This configuration file is custom generated for NewRelic Administration
+#
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  # Use NEW_RELIC_LICENSE_KEY env variable to specify the license key
+
+  # Your application name. Renaming here affects where data displays in New
+  # Relic.  For more details, see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/renaming-applications
+  # Use NEW_RELIC_APP_NAME env variable to customize the application name
+  app_name: "Open Food Network"
```

Comment:
I have one question:
Will all the instances share the same key? and NEW_RELIC_APP_NAME will variate depending on the instance location. for example:  "Open Food Network (AU)", "Open Food Network (UK)" ?

Reply:
That's a good question. In the past we would have set up separate keys but if New Relic supports it then we should use the same key, I think. That we we have all the information in one place.

Reply:
My understanding is we should be able to use the same license key for all the instances, and set up a different name for each instance so we can differentiate the data. See below: 
![New_relic](https://github.com/openfoodfoundation/openfoodnetwork/assets/40413322/0371f530-1cac-461c-ba5b-6a323c9799e1)
There is the possibility to create extra license key so we could potentially have a different license key for each instance, all reporting to the same New Relic account. Or have a different user with it's own license key for each instance, but that's probably not needed.  It's recommended to create extra license for security reason (key rotation) .
More info here: https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/

Reply:
Okay, I think that we should roll out with the same key and we can rotate later.